src/main/java/cn/nukkit/command/args/basic/BoolArgument.java

```diff
@@ -0,0 +1,8 @@
+package cn.nukkit.command.args.basic;
+
+public class BoolArgument extends EnumArgument<Boolean> {
+
+    public BoolArgument() {
```

Comment:
Make arguments as enums class

Reply:
why?

Reply:
Then you are not limited to the set of vanilla parameters. It's possible to have arguments like Player, Level, Block etc directly

Reply:
These classes will also have custom parsing, when i can figure it out. I am getting most of my ideas from Mojang's own [brigadier](https://github.com/Mojang/brigadier)

Reply:
@lukeeey just implement brigadier.

Reply:
I think you’ll find I already made a PR for that and it’s simply harder than you make think. I could fork brigadier and add all the features I need but that seems liked the harder thing to do.

Additionally, I’m not a fan of how arguments are chained together because it ends up looking quite messy and long due to the indentation.